### ISSUE TITLE
Data insertion in postgres UUID column fails. Add support for rust native uuid::Uuid data type

### ISSUE THREAD

**Issue by PPakalns** [rbatis/rbatis#59]
When struct field has type `uuid::Uuid` it doesn't correctly translate to native postgres uuid type.

uuid data type is represented as string instead.

See the following error from database:
```
error returned from database: column "id" is of type uuid but expression is of type text'
```

Additionally add support for CRUDEnable IdType = uuid::Uuid.

**Comment by PPakalns** [comment]
Tested with `String` field. It seems that even String insertion in postgres UUID field doesn't work.

**Comment by zhuxiujia** [comment]
> When struct field has type `uuid::Uuid` it doesn't correctly translate to native postgres uuid type.
> 
> uuid data type is represented as string instead.
> 
> See the following error from database:
> 
> ```
> error returned from database: column "id" is of type uuid but expression is of type text'
> ```
> 
> Additionally add support for CRUDEnable IdType = uuid::Uuid.

i  try fix this issues, you can add cargo.toml   dep   
```rust
 rbatis = { git = "https://github.com/rbatis/rbatis.git"}
```

```rust
    #[async_std::test]
    pub async fn test_postgres_uuid() {
        fast_log::init_log("requests.log", 1000, log::Level::Info, None, true);
        let rb = Rbatis::new();
        rb.link("postgres://postgres:123456@localhost:5432/postgres").await.unwrap();

        #[crud_enable(formats:id:{}::uuid)]
        #[derive(Clone, Debug)]
        pub struct BizUuid {
            pub id: Option<Uuid>,
            pub name: Option<String>,
        }
        let uuid = Uuid::from_str("df07fea2-b819-4e05-b86d-dfc15a5f52a9").unwrap();
        //make table
        rb.exec("", "CREATE TABLE biz_uuid( id uuid, name VARCHAR, PRIMARY KEY(id));").await;
        //insert table
        rb.save("", &BizUuid { id: Some(uuid), name: Some("test".to_string()) }).await;
        //query table
        let data: BizUuid = rb.fetch_by_id("", &uuid).await.unwrap();
        println!("{:?}", data);
        //remove table
        let uuid=Uuid::from_str("df07fea2-b819-4e05-b86d-dfc15a5f52a9").unwrap();
        rb.remove_by_id::<BizUuid>("",&uuid).await;
    }
2020-12-14 00:53:57.009017800 +08:00 INFO rbatis::plugin::log - [rbatis] [] Exec  ==> CREATE TABLE biz_uuid( id uuid, name VARCHAR, PRIMARY KEY(id));
2020-12-14 00:53:57.016153600 +08:00 INFO rbatis::plugin::log - [rbatis] [] Exec  ==> INSERT INTO biz_uuid (id,name) VALUES ($1::uuid,$2)
                                                                [rbatis] [] Args  ==> ["df07fea2-b819-4e05-b86d-dfc15a5f52a9","test"]
2020-12-14 00:53:57.019584500 +08:00 INFO rbatis::plugin::log - [rbatis] [] RowsAffected <== 0
2020-12-14 00:53:57.019803300 +08:00 INFO rbatis::plugin::log - [rbatis] [] Query ==> SELECT id,name FROM biz_uuid WHERE id = $1::uuid
                                                                [rbatis] [] Args  ==> ["df07fea2-b819-4e05-b86d-dfc15a5f52a9"]
2020-12-14 00:53:57.023173600 +08:00 INFO rbatis::plugin::log - [rbatis] [] ReturnRows <== 1
2020-12-14 00:53:57.023376700 +08:00 INFO rbatis::plugin::log - [rbatis] [] Exec  ==> DELETE FROM biz_uuid WHERE id = $1::uuid
                                                                [rbatis] [] Args  ==> ["df07fea2-b819-4e05-b86d-dfc15a5f52a9"]
2020-12-14 00:53:57.029783900 +08:00 INFO rbatis::plugin::log - [rbatis] [] RowsAffected <== Ok(
    DBExecResult {
        rows_affected: 1,
        last_insert_id: None,
    },
)
BizUuid { id: Some(df07fea2-b819-4e05-b86d-dfc15a5f52a9), name: Some("test") }
```

**Comment by zhuxiujia** [comment]
fix in   rbatis v1.8.47
